genume/view/main.py

```diff
@@ -121,7 +121,7 @@ def set_button_location(s, b, allocation):
 
         refresh_button = self.generate_refresh_button()
         main_view.add_overlay(refresh_button)
-        main_view.set_overlay_pass_through(refresh_button, True)
+        # main_view.set_overlay_pass_through(refresh_button, True)
```

Comment:
This based on the doc should break the app by making everything but the refresh button clickable, but when I run it everything works.

We should wait until #62 is closed first (I will look into it in the next days)